src/main/webapp/cdn/blockly/generators/propc/console.js

```diff
@@ -122,14 +122,17 @@ Blockly.Blocks.console_move_to_row = {
 
 Blockly.propc.console_newline = function () {
     return 'print("\\r");';
+    Blockly.propc.serial_terminal_ = true;
```

Comment:
This won't get executed as it is below the return statement. ( same for console_clear)